README

```diff
@@ -294,6 +301,15 @@ The default behaviour can be changed with the following directives:
 
 	Default: 5000 (== 5 s).
 
+    LookupArgumentName
+
+    Name of the variable to look for in request query string. The
+    value from query string (if there is any) is then sent to SSSD
+    together with the certificate. This is usefull when single
+    certificate is assigned to multiple user accounts.
+
+    Default: username
```

Comment:
Probably not. Do I need to handle somehow the situation when the directive is missing in configuration?